<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BddJoueur</class>
 <widget class="QDialog" name="BddJoueur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>670</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Statut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter joueur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtUpdate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier joueur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtDelete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer joueur</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>451</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtLoad">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>630</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Table</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditId"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNom"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPrenom"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Score</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditScore"/>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>161</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Liste des joueurs:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
